packages/beta/src/__tests__/api/documentsApi.int.spec.ts

```diff
@@ -34,4 +34,16 @@ describe.skip('documents api', () => {
     const response = await client.documents.pipelines.list();
     expect(response.items.length).toBeGreaterThan(0);
   });
+  test('fetch preview', async () => {
+    const mediaTypePDF = 'application/pdf';
+    const documents = await client.documents.list({
+      limit: 1,
+      filter: { mimeType: { equals: mediaTypePDF } },
+    });
+    expect(documents.items.length).toEqual(1);
+    const document = documents.items[0];
+
+    await client.documents.preview.documentAsPdf(document.id);
+    await client.documents.preview.documentAsImage(document.id, 0);
```

Comment:
Should we assert that the response is as expected here? Not just the status code, but the contents.